app/components/spark-line/component.js

```diff
@@ -1,28 +1,32 @@
 import Ember from 'ember';
-import { formatPercent, formatMib, formatKbps } from 'ui/utils/util';
+import {
+  formatPercent, formatMib, formatKbps
+}
+from 'ui/utils/util';
 
 const formatters = {
-  value: (value) => { return value; },
+  value: (value) => {
+    return value;
+  },
   percent: formatPercent,
   mib: formatMib,
   kbps: formatKbps
 };
 
-const DOT_SIZE = 2;
-
 export default Ember.Component.extend({
   data: null,
   tagName: 'span',
   classNames: ['spark-line'],
-  attributeBindings: ['cssSize:style','tooltip'],
+  attributeBindings: ['cssSize:style', 'tooltip'],
 
   width: null,
   height: 20,
   min: 0,
   max: null,
-  interpolation: 'step-after',
+  interpolation: 'linear',
```

Comment:
I can change this back, i had changed it to match the mocks. The bigger charts are also linear in the mocks do we want to keep them stepped?